make/common/NativeCompilation.gmk

```diff
@@ -1165,10 +1165,12 @@ define SetupNativeCompilationBody
       $1_LD_OBJ_ARG := `cat $$($1_OBJ_FILE_LIST)`
     endif
 
-    # If we are building static library, 'AR' on macosx may not support @-file.
+    # If we are building static library, 'AR' on macosx/aix may not support @-file.
     ifeq ($$($1_TYPE), STATIC_LIBRARY)
       ifeq ($(call isTargetOs, macosx), true)
         $1_LD_OBJ_ARG := `cat $$($1_OBJ_FILE_LIST)`
+      else ifeq ($(call isTargetOs, aix), true)
```

Comment:
you could spare the `else ifeq` branch but rather use `ifeq ($(call isTargetOs, macosx aix), true)`

Reply:
Good idea. Will make the change now.